src/generator/snapshots/tailcall__generator__from_proto__test__from_proto.snap

```diff
@@ -36,6 +36,11 @@ input NEWS_NEWS_ID @tag(id: "news.NewsId") {
   id: Int
 }
 
+enum NEWS_STATUS @tag(id: "news.Status") {
+  DRAFT
+  PUBLISHED
+}
```

Comment:
Add tests for messages inside of messages.

Reply:
news.proto already contains message inside of message https://github.com/tailcallhq/tailcall/pull/1747/files#diff-771adfaaae91c7816aaf69af02c57749bfaaaa74e3db399e055104a8a1a6de23R26-R33

is it enough?